src/Servers/Kestrel/shared/KnownHeaders.cs

```diff
@@ -40,23 +40,37 @@ public class KnownHeaders
         .Select(h => h.Name)
         .ToArray();
 
-    // These headers are excluded from generated IHeadersDictionary implementaiton.
+    // These headers are excluded from generated IHeadersDictionary implementation.
     public static readonly string[] NonPublicHeaderNames = new[]
     {
         HeaderNames.DNT,
         InternalHeaderNames.AltUsed
     };
 
-    public record InternalHeader(string Identifier, string Name, bool IsPseudoHeader = false);
+    public class InternalHeader
```

Comment:
I don't think this "shared" means "with runtime", but @BrennanConroy might be able to confirm.

Reply:
I don't understand the question

Reply:
We have some code that is shared with the runtime repo.  This isn't part of it, correct?

Reply:
Right. That code is in https://github.com/dotnet/aspnetcore/tree/main/src/Shared/runtime